ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml

```diff
@@ -1255,6 +1255,15 @@ CHPM:
   implements:
   - MONITORING
 
+EFSM:
```

Comment:
nit: adjust to `ESFM`, so as not to confuse with `EF` for exhaust fan.

Reply:
Sure. Done the required changes.

Reply:
Done the required changes.